assets/src/components/ai/chatbot/AISuggestFix.tsx

```diff
@@ -121,15 +123,15 @@ function FixPr({
         >
           View PR
         </Button>
-      ) : (
+      ) : personaConfiguration?.ai?.pr !== false ? (
```

Comment:
we should also respect this if the top level `all` flag is set to true.

Reply:
Done. I have added generic typesafe method to check that.